dist/php/businessEmailValidator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class BusinessEmailValidator
+{
+
+	/**
+	* Used to load list of blocked domains from local JSON file.
+	* @access private
+	* @return array
+	*/
+	private function listBlocked()
+	{
+		$path = 'src/freeEmailService.json';
+		file_exists($path) or die("Error: file 'freeEmailService.json' not found!");
+		return (array) json_decode(file_get_contents($path), true);			
+	}
+
+	/**
+	* Used to check if provided email address is in list of blocked domains
+	* @access public
+	* @param string $email
+	* The email address we want to check
+	* @example 'john@doe.com'
+	* @return bool
+	*/
+	public function isBlocked($email)
```

Comment:
The return type is bool but your function calls `die x` `return x`  or `echo x`
You should only return true or false whenever the email is valid or not